sdk/servicebus/azure-servicebus/samples/async_samples/topic_subscription_with_rule_operations.py

```diff
@@ -0,0 +1,159 @@
+#!/usr/bin/env python
+
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+"""
+Example to show receiving batch messages from a Service Bus Subscription under specific Topic asynchronously.
```

Comment:
nit: async sample should have "_async" suffix in the file name.

also counterpart in sync sample?